### DISCUSSION TITLE
Existing EPPlus Open Source Fork as a potential replacement

### DISCUSSION THREAD

**Discussion by ubidev** [dfinke/ImportExcel#1595] [Ideas]
Hi, after finding no trace of this project mentionned in the past issues and discussions I thought I'd share that I have found the following: [https://github.com/PerfectXL/PerfectXL-EPPlus](https://github.com/PerfectXL/PerfectXL-EPPlus)
Anyone think this could help ImportExcel considering that EPPlus has been a paid product since a while now?

Have a great time everyone :)

**Comment by dfinke** [comment]
Thanks @ubidev, looks interesting. Unfortunately, it has not been updated in years. Other than ther performance improvements, can't tell what else was done.

![image](https://github.com/dfinke/ImportExcel/assets/67258/bf3c3dfc-543d-4cb8-a969-0133956b3602)

**Reply by ubidev** [reply]
I was hoping that the changes implemented since the fork would help some of the current needs of ImportExcel.

I'll soon fork it and will inspect the codebase as to see if I develop any interest in contributing to it as a way to give back to this community. Maybe we could consider to port it inside this project to avoid a similar situation in the future. 

What do you think Sir?

**Reply by dfinke** [reply]
Thanks @ubidev. I like the idea, not sure I'm up for maintenance support.

When the paid version of EPPlus came out I did spike with it and ImportExcel. For example, they changed the zero based array to one base.

Took a bunch of time to sort some of those issues and more.

EPPlus forged ahead with slicers etc. 

If you take a look at the replacement, be interested in what you find and think. Happy to discuss options.

**Reply by ubidev** [reply]
Great then !

**Comment by ubidev** [comment]
Oh and I forgot to ask if forking/integrating EEplus in ImportExcel was ever considered after it went commercial and if yes, why wasn't it done?
Thanks in advance and have a great weekend

**Comment by dfinke** [comment]
No I did not fork EPPlus and try.
I only tried the new EPPlus they delivered.
